tests/lightning_tensor/test_gates_and_expval.py

```diff
@@ -73,7 +73,7 @@ def circuit_ansatz(params, wires):
         qml.matrix(qml.PauliX([wires[1]])), control_wires=[wires[0]], wires=wires[1]
     )
     qml.DiagonalQubitUnitary(np.array([1, 1]), wires=wires[2])
-    qml.MultiControlledX(wires=[wires[0], wires[1], wires[3]], control_values=[wires[0], wires[1]])
```

Comment:
Thanks @JerryChen97 . Is the target wire  `wires[-1]` by default?

Reply:
Yes, and we can have https://docs.pennylane.ai/en/stable/code/api/pennylane.MultiControlledX.html as ref here.

Reply:
Please add the `gpu-runner` tag to this PR to ensure the change will be tested

Reply:
Please re-trigger the CIs to ensure the changes made will be tested.